graylog2-web-interface/src/pages/IndexSetPage.jsx

```diff
@@ -132,12 +132,16 @@ const IndexSetPage = React.createClass({
     if (this.state.indexerOverview && this.state.indexDetails.closedIndices) {
       const deflectorInfo = this.state.indexerOverview.deflector;
 
+      let activeIndex = ".";
+      if (indexSet.writable) {
+        activeIndex = ", current write-active index is <i>{deflectorInfo.current_target}</i>";
```

Comment:
This renders as:
![image](https://cloud.githubusercontent.com/assets/52014/23791658/23e2fd2a-0584-11e7-9e1e-11edf3a72abc.png)

I would wrap the text in a `<span>` instead of making it a string. Note that a template string would not work either, because of the `<i>` tag. If the span is multiline please also wrap it in `()`.